prelude/src/core/language/Iterator.sk

```diff
@@ -354,6 +365,23 @@ mutable class MapIterator<T, +U>(
   }
 }
 
+// See Iterator#mapWhile()
+mutable class MapWhileIterator<T, +U>(
+  private base: mutable Iterator<T>,
+  private f: T -> ?U,
+) extends Iterator<U> {
+  readonly fun sizeHint(): ?Int {
+    this.base.sizeHint()
+  }
+
+  mutable fun next(): ?U {
+    this.base.next() match {
+    | Some(x) -> this.f(x)
+    | None() -> None()
+    }
+  }
+}
+
```

Comment:
No need for this since you're using `yield`

Reply:
Doh! Leftover from previous iteration, removed in https://github.com/SkipLabs/skdb/pull/252/commits/a9b3bd1b7ad73126a4493b73c9df931629ec3584